docusaurus/docs/actors/relay_miner.md

```diff
@@ -0,0 +1,32 @@
+---
+title: RelayMiner
+sidebar_position: 5
+---
+
+# RelayMiner <!-- omit in toc -->
+
+- [Overview](#overview)
+- [Configuration](#configuration)
+- [CLI](#cli)
+
+## Overview
+
+An `RelayMiner` is responsible for for relaying requests between a client dApp
```

Comment:
I think you wanted to say: Relaying requests between the `AppGate Server` and the `Service`